.github/ct.yaml

```diff
@@ -0,0 +1 @@
+target-branch: main
```

Comment:
I would add a comment saying the purpose of the file. 
`That is the name of the target branch used to identify changed charts` I guess we need that now since we moved from master to main and the default was master

Reply:
Solved in [#bb085fd](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/bb085fdae4dd77d656cf0a01cf238efca5f133e3)